packages/playground-examples/copy/pt/JavaScript/Functions with JavaScript/Generic Functions.ts

```diff
@@ -0,0 +1,100 @@
+// Tipos genéricos provêm uma forma de usar Tipos como variáveis em outros
+// tipos. Meta.
+
+// Tentaremos manter este exemplo leve. Você pode fazer
+// muita coisa com tipos genéricos e é provável que veja algum código bem
+// complicado usando tipos genéricos em algum ponto - mas isso
+// não significa que tipos genéricos são complicados.
+
+// Vamos começar com um exemplo onde envolveremos um objeto de entrada
+// em um array. Apenas nos importaremos com uma só variável neste
+// caso: o tipo que foi passado como argumento:
+
+function envolverEmArray<Tipo>(entrada: Tipo): Tipo[] {
+  return [entrada];
+}
+
+// Nota: é comum ver Tipo ser referido como T. Isso é
+// culturalmente similar à como as pessoas usam i em um loop for
+// para representar índice. T normalmente representa Tipo, então
```

Comment:
```suggestion
// para representar índice. T normalmente representa Tipo (Type), então
```

Reply:
In this case, should we leave `Type` untranslated? It seems a bit off to refer back to the original word here given that it's translated in every instance of the document.

Reply:
T is an abbreviation for `Type`, which coincidentally translates to `Tipo` and has the same abbreviation.
My thought process is that it is better to be clear that it comes from the English word.

Maybe use `Tipo (Type em inglês)`

Reply:
I agree with @danilofuchs on this. I think we should have both